server/tracer/src/main/java/org/apache/accumulo/tracer/TraceServer.java

```diff
@@ -169,23 +170,25 @@ public void span(RemoteSpan s) throws TException {
         if (timeMutation != null)
           writer.addMutation(timeMutation);
       } catch (MutationsRejectedException exception) {
-        log.warn("Unable to write mutation to table; discarding span. set log level to DEBUG for span information and stacktrace. cause: " + exception);
+        log.warn("Unable to write mutation to table; discarding span. set log level to DEBUG for span information and stacktrace. cause: {}", exception,
```

Comment:
No problem, will do.  I now remember grep'ing the code for occurrences of log string concatenation that could be replaced by slf4j logging but quickly gave up, haha.